.github/workflows/release.yaml

```diff
@@ -203,22 +205,19 @@ jobs:
     steps:
 
       - name: set the release version (tag)
-        if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
I think we need to do more than just remove this line, since there won't (in general) be a tag for `main`.  I was expecting something like this: https://github.com/fermyon/spin/blob/3e75c42202effe68ccfd09726ac6b572e6d96dab/.github/workflows/release.yml#L66-L74

Reply:
You're right. We're doing the right thing in the `release` job but not in the `checksums` job. I've pushed a change that fixes this.